content/en/docs/languages/java/jmx.md

```diff
@@ -0,0 +1,341 @@
+
+   ```sh
+   curl -L -O https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar
+   ```
+
+2. Run your application with the agent and enable JMX metrics:
+
+   ```sh
+   java -javaagent:opentelemetry-javaagent.jar \
+     -Dotel.jmx.enabled=true \
+     -jar myapp.jar
+   ```
+
+### As a Standalone Library
+
+You can also use the JMX metrics library as a standalone dependency:
+
+Add the following dependency to your project:
+
+**Maven (`pom.xml`):**
+
+```xml
+<dependency>
+  <groupId>io.opentelemetry.instrumentation</groupId>
+  <artifactId>opentelemetry-jmx-metrics</artifactId>
+  <version>{{% param vers.instrumentation %}}-alpha</version>
+</dependency>
+```
+
+**Gradle (`build.gradle.kts`):**
+
+```kotlin
+dependencies {
+  implementation("io.opentelemetry.instrumentation:opentelemetry-jmx-metrics:{{% param vers.instrumentation %}}-alpha")
+}
+```
+
+## Configuration
+
+JMX metrics can be collected in two modes:
+
+- **Java Agent mode** - Collects metrics from the same JVM running the agent
+- **Scraper mode** - Collects metrics from a remote JVM via JMX remote connections
+
+### Java Agent Mode Configuration
+
+When using the OpenTelemetry Java Agent, configure JMX metrics using these
+properties:
+
+| System Property                  | Environment Variable             | Description                                     | Default |
+| -------------------------------- | -------------------------------- | ----------------------------------------------- | ------- |
+| `otel.jmx.enabled`               | `OTEL_JMX_ENABLED`               | Enable JMX metric collection                    | `false` |
+| `otel.jmx.target.system`         | `OTEL_JMX_TARGET_SYSTEM`         | Predefined metric set to use                    | none    |
+| `otel.jmx.interval.milliseconds` | `OTEL_JMX_INTERVAL_MILLISECONDS` | Collection interval in milliseconds             | `10000` |
+| `otel.jmx.groovy.script`         | `OTEL_JMX_GROOVY_SCRIPT`         | Path to custom Groovy script for metric mapping | none    |
+
+### Scraper Mode Configuration
+
+When using the standalone JMX Metric Gatherer (scraper), configure using these
+properties (note: `otel.jmx.enabled` is not needed):
+
+| System Property                  | Environment Variable             | Description                                     | Default    |
+| -------------------------------- | -------------------------------- | ----------------------------------------------- | ---------- |
+| `otel.jmx.service.url`           | `OTEL_JMX_SERVICE_URL`           | JMX service URL for remote JVM connection       | (required) |
+| `otel.jmx.target.system`         | `OTEL_JMX_TARGET_SYSTEM`         | Predefined metric set to use                    | none       |
+| `otel.jmx.interval.milliseconds` | `OTEL_JMX_INTERVAL_MILLISECONDS` | Collection interval in milliseconds             | `10000`    |
```

Comment:
With JMX Scraper, we reuse the SDK metric export interval `otel.metric.export.interval` thus this option is not relevant
```suggestion
```